gradle/tests.gradle

```diff
@@ -114,21 +120,14 @@ test {
   useJUnitPlatform()
 }
 
-tasks.test.dependsOn(buildZkevmBin)
-
-tasks.register('unitTests', Test) {
-  description = 'Runs unit tests.'
-
-  minHeapSize = "4g"
-  maxHeapSize = "8g"
-  maxParallelForks = Runtime.getRuntime().availableProcessors().intdiv(2) ?: 1
-
+tasks.test.configure {
   useJUnitPlatform {
-    excludeTags("AcceptanceTest")
-    exclude('**/**ReferenceTest*')
+    excludeTags("nightly")
```

Comment:
why don't you exclude the referenceTest anymore?
Are they excluded through some other mechanism?

Reply:
If you run `./gradlew :test` it will run all tests except the ones tagged "nightly". But we use `./gradlew :arithmetization:test` in the CI for unit tests, which runs tests under the `arithmetization` module. Reference test are present under the `reference-test` module so excluding them is not required if we only use `./gradlew :arithmetization:test`

Reply:
You can run reference tests in the following ways:
- `./gradlew :reference-test:test`
- `./gradlew referenceGeneralStateTests`